### DISCUSSION TITLE
Is it possible to install Scratch Addons on Edge for Android?

### DISCUSSION THREAD

**Discussion by Dhuls-Scratch** [ScratchAddons/ScratchAddons#3440] [Help] [answered]
I would prefer to not have to use a beta browser, but I might if I have to.

**Comment by apple502j** [comment]
Scratch Addons is not tested on neither Edge nor Android devices; I am not sure if it works. Just try it and if it fails, get a laptop.

**Comment by Samq64** [comment, marked as answer]
Related: #3255

In general I think browser extensions are just hard to install on mobile. Firefox supports only a few, and Chrome & Edge don't support any at all as far as I can tell. I think the only way to install the extensions you want on mobile is to use the developer versions of mobile browsers or use [Kiwi](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser)
